pkg/analyze/cluster_pod_statuses_test.go

```diff
@@ -0,0 +1,266 @@
+package analyzer
+
+import (
```

Comment:
Good stuff with the tests!

Nit pick: Could you add/modify tests to use `!==`,  `==`, `===` and `=` conditionals for completeness? e.g `=== CrashLoopBackOff`